cmd/internal/kube/kube.go

```diff
@@ -12,3 +12,45 @@ var Cmd = &cobra.Command{
 		"kubernetes",
 	},
 }
+
+func init() {
+
+	// `kube` command level flags
+	Cmd.PersistentFlags().StringVar(
+		&insightsProjectID,
+		"project",
+		"",
+		"Your Postman Insights project ID.")
+	_ = Cmd.MarkFlagRequired("project")
+
+	// `kube inject` command level flags
+	injectCmd.Flags().StringVarP(
+		&injectFileNameFlag,
+		"file",
+		"f",
+		"",
+		"Path to the Kubernetes YAML file to be injected. This should contain a Deployment object.",
+	)
+	_ = injectCmd.MarkFlagRequired("file")
```

Comment:
I don't understand why the inject-specific commands got moved into a file where neither injectCmd nor the flag variables are defined.  Is there an ordering problem with the init functions?  Or a problem with having more than one, maybe?

Reply:
I have moved it under kube since this `init()` function also has parts where we add the new subcommands L54-55 and the main `--project` flag, which is common for all 3 subcommands.
I will split the `init()` function into 3 functions for each file.